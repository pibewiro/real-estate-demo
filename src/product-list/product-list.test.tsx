import { render } from '@testing-library/react';
import ProductList from './product-list';

describe('test', () => {
  it('sub test 1', () => {
    const a = render(<ProductList />);
    expect(a.container.firstChild).toMatchSnapshot();
  });
});
