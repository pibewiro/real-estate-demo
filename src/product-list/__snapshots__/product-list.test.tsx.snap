// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test sub test 1 1`] = `
<div>
  <h1>
    Product List1
  </h1>
</div>
`;
